action.yml

```diff
@@ -25,6 +31,13 @@ inputs:
     description: 'Comma separated list of rules to exclude from Sprocket check.'
     required: false
     default: ''
+  # Options for `validate-inputs` subcommand
+  wdl_file:
+    description: "Path to the WDL document for which to validate inputs."
+    required: false
+  inputs_file:
+    description: "Path to the JSON file containing inputs to validate."
+    required: false
```

Comment:
This should probably accept an array of each so that multiple checks are possible with the same action.

Reply:
Do you have an example of how this is going to work in CI? I don't really want to support arrays in bash and the underlying tool doesn't accept an array. I'm curious if you can accomplish processing multiple files with GitHub Actions or not.

Reply:
My thinking is for the workflows repo, we want to check all our `reference/` workflows with their respective "production value" JSONs. With the API you've written here, that would require a dedicated action for each reference workflow. That would be annoying to maintain.

1. The CI logs would get cluttered with a bunch of new actions.
2. Adding or removing new workflow checks would require writing/removing entire actions.

It'd be much simpler if we could have one action that checks all our reference workflows.